README.md

```diff
@@ -249,4 +175,4 @@ Report it here: https://github.com/datamade/nyc-councilmatic/issues
 
 ## Copyright
 
-Copyright (c) 2017 DataMade. Released under the [MIT License](https://github.com/datamade/nyc-councilmatic/blob/master/LICENSE).
+Copyright (c) 2019 DataMade. Released under the [MIT License](https://github.com/datamade/nyc-councilmatic/blob/master/LICENSE).
```

Comment:
Link to this license: https://github.com/datamade/la-metro-councilmatic/blob/master/LICENSE